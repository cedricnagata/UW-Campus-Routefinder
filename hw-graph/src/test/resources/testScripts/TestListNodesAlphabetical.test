# Create a graph
CreateGraph Graph

# Adds two nodes to the graph
AddNode Graph nodeB
AddNode Graph nodeA

# List the Nodes in the graph
ListNodes Graph