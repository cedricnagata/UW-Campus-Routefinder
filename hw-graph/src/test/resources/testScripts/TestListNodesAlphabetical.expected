# Create a graph
created graph Graph

# Adds two nodes to the graph
added node nodeB to Graph
added node nodeA to Graph

# List the Nodes in the graph
Graph contains: nodeA nodeB